metrics/time/README.md

```diff
@@ -0,0 +1,10 @@
+# Time metrics tests
+
+This section includes metrics tests focused on measure the time interval[s] of a determined
+number of containers executing a defined workload. This kind of metrics tests could provide
+a general overview about the time spent by a flow, however the use of these should be
+complemented with more tools in order to find the hot spot.
```

Comment:
in order to find the hot spot, these tests should be used in combination with more tools.